_scripts/instruction-widget/main.js

```diff
@@ -90,6 +81,15 @@ InstructionWidget = (function() {
     }
   };
 
+  // Users used to be able to link to an instruction set with an anchor link.
+  // We can redirect them to a standalone page.
+  redirect_anchor = function() {
+    var params = window.location.hash.replace('#', '').split('-');
+    if (params.length === 2) {
+      jump(params[0], params[1]);
```

Comment:
This rewrote the url for me but didn't render any instructions.

http://localhost:4000/#debianjessie-nginx